app/src/main/res/layout/fragment_new_detail.xml

```diff
@@ -1,58 +1,83 @@
 <?xml version="1.0" encoding="utf-8"?>
```

Comment:
Check this xml file, and add an empty line between every different component to make this more readable.